### PR TITLE
Libxrandr update to 1.5.2, use clang linker

### PR DESCRIPTION
**Pull request by satmandu**
Part of https://github.com/skycocker/chromebrew/pull/4450

Works properly:
- [x] x86_64

### CODE REVIEW

File: packages/libxrandr.rb [hunk 1]
```diff
@@ -3,28 +3,19 @@
...
-     x86_64: '6a37502061445761f860acefaddd5bf06b7c64b4b5e468f4e3902e67baaef05e',
-  })
 
   depends_on 'libxrender'
   depends_on 'libx11'
+  depends_on 'llvm' => ':build'
 
   def self.build
+    ENV['CFLAGS'] = "-fuse-ld=lld"
+    ENV['CXXFLAGS'] = "-fuse-ld=lld"
```
Comment (uberhacker) [inline packages/libxrandr.rb:18]: Does this break arm builds?

File: packages/libxrandr.rb [hunk 2]
```diff
@@ -3,28 +3,19 @@
...
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```
Comment (uberhacker) [inline packages/libxrandr.rb:19]: Use `#{CREW_OPTIONS}`?

File: packages/libxrandr.rb (hunk 1 above)
Comment (satmandu) [reply packages/libxrandr.rb:18]: gcc10...

**Comment by uberhacker** [comment]
Merge conflicts?

**Comment by satmandu** [comment]
done, but I'm wondering if any llvm requirement should also have a gcc10 requirement as well.

**Pull request merged**